flow360/component/resource_base.py

```diff
@@ -112,6 +113,33 @@ def wrapper(obj, *args, **kwargs):
     return wrapper
 
 
+class AssetMetaBaseModelV2(pd_v2.BaseModel):
```

Comment:
Had to go with V2 pydantic since we cannot mix fields of V1 and V2 in same model.